internal/cmd/edit.go

```diff
@@ -34,7 +35,15 @@ func newEditCmd() *editCmd {
 			}
 
 			log.Printf("%s %s\n", editor, tmp)
-			edit := exec.Command(editor, tmp)
+
+			editorArgs := []string{tmp}
+			if strings.ContainsAny(editor, " ") {
+				editorParts := strings.Split(editor, " ")
+				editor = editorParts[0]
+				editorArgs = append(editorArgs, editorParts[1:]...)
+			}
+
+			edit := exec.Command(editor, editorArgs...)
```

Comment:
I think we can make this a bit nicer:

```go
editor := strings.Fields(os.Getenv("EDITOR"))
if len(editor) == 0 {
 			return fmt.Errorf("no $EDITOR set")
}

cmd := editor[0]
var args []string
if len(editor) > 1 {
  args = append(args, editor[1:])
}
args = append(args, tmp)

edit := exec.Command(cmd, args)
// ...
```

wdyt?

Reply:
Nice, I didn't know `strings.Fields` yet. Thanks for the suggestion, I'll update the PR in just a moment. I have to rename the variable though since `cmd` and `args` are the parameter names of the function already :)